composer.json

```
@@ -29,16 +29,16 @@
         "illuminate/support": "^10.0",
         "illuminate/view": "^10.0",
         "moneyphp/money": "^3.2|^4.0",
-        "nesbot/carbon": "^2.0",
+        "nesbot/carbon": "^2.67",
         "spatie/url": "^1.3.5|^2.0",
-        "symfony/http-kernel": "^6.0",
+        "symfony/http-kernel": "^6.2",
         "symfony/polyfill-intl-icu": "^1.22.1"
     },
     "require-dev": {
-        "mockery/mockery": "^1.0",
-        "orchestra/testbench": "^8.0",
-        "phpstan/phpstan": "^1.10",
-        "phpunit/phpunit": "^9.0"
+        "mockery/mockery": "^1.5.1",
+        "orchestra/testbench": "^8.14",
+        "phpstan/phpstan": "1.10.41",
```

```suggestion
        "phpstan/phpstan": "^1.10",
```

Made a mistake locking phpstan deps while debug memory leak issue.